Adapter_oM/enums/PushType.cs

```diff
@@ -45,7 +46,8 @@ public enum PushType //add error message if not picked up --> compliance check f
         [Description("For all objects being Pushed, identifies their type, calls Delete to remove all of those types, then it Creates them.")]
         DeleteThenCreate,
         [Description("If this is chosen, then the m_AdapterSettings.DefaultPushType is picked.")]
-        AdapterDefault
+        AdapterDefault,
+        CreateNonExisting
```

Comment:
Move up just below CreateOnly

Reply:
Also, please add Add same description provided above the enum. 
```cs
...
        [Description("Uses only the Create CRUD method to export the objects. This may create duplicates if the object already exists.")]
        CreateOnly,
        [Description("------>Add same description provided above the enum<------")]
        CreateNonExisting,
        [Description("Uses only the Update CRUD method to update the objects in the external software.")]
        UpdateOnly,
 ...
```

Reply:
Not afraid this could skrew up pre-exiting uses? Might have dropdowns with enums linking over to the wrong thing if we insert it in the middle of the list?